### DISCUSSION TITLE
Modifying the link of the Draw.io instance that the Google Docs add-on points to

### DISCUSSION THREAD

**Discussion by brnpl89** [jgraph/drawio#3981] [General]
We have tested the drawio add-on for Google Docs and we found it useful. However, we want to modify the link for the Drawio instance that the add-on points to with our own self-hosted Drawio instance instead of the public one when we click on the "New Diagram" or "Edit Diagram" shortcuts or visualize the link for a diagram we inserted. Is it possible to do so?

**Comment by davidjgraph** [comment]
No, that isn't possible.
